vmware_aria_operations_integration_sdk/adapter_configurations/adapter_templates/java/sample_adapter/src/main/java/Adapter.java.template

```diff
@@ -1,100 +1,209 @@
+
```

Comment:
This looks like the `new_adapter`, but seems to be listed under `sample_adapter`. I don't see the sample adapter in this PR?